omnisend/class-omnisend-core-bootstrap.php

```diff
@@ -40,6 +44,8 @@
 class Omnisend_Core_Bootstrap {
 
 	public static function load(): void {
+		add_filter( 'cron_schedules', 'Omnisend_Core_Bootstrap::cron_schedules' ); // phpcs:ignore
```

Comment:
Please add additional comment why lint is ignored

Reply:
Added